amplify-ui-geo-explore/src/App.js

```diff
@@ -0,0 +1,130 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: MIT-0
+
+import { useState, useEffect } from "react";
+import Amplify, { Auth } from "aws-amplify";
+import { LocationClient } from "@aws-sdk/client-location";
+import { IDENTITY_POOL_ID, REGION, MAP, PLACE } from "./configuration";
+import PlacesLayer from "./components/places/PlacesLayer";
+import RoutesLayer from "./components/routes/RoutesLayer";
+import { MapView } from "@aws-amplify/ui-react";
+import { NavigationControl } from "react-map-gl";
+import WebMercatorViewport from "@math.gl/web-mercator";
+import "@aws-amplify/ui-react/styles.css";
+import "./index.css";
+
+// Configuring Amplify Geo with existing Amazon Cognito and Amazon Location Service information
+// Fill values in configuration.js
+Amplify.configure({
```

Comment:
This works and the demo works fine, but I get errors in the console (see screenshot below):

![Screenshot 2022-05-02 at 10 29 16](https://user-images.githubusercontent.com/7353869/166251704-19c6e4de-d321-419c-b8aa-872e07560702.png)

Can we look into it and see if we can remove them?

Reply:
[Maybe helpful](https://docs.amplify.aws/lib/client-configuration/configuring-amplify-categories/q/platform/js/#general-configuration).

Reply:
This is an existing issue with Amplify Auth https://github.com/aws-amplify/amplify-js/issues/9845. I have left notes in the README of the sample app describing this issue and will remove it when Amplify has the fix ready.